spec/helpers/claims_helper_spec.rb

```diff
@@ -44,6 +61,8 @@
         ["Teacher reference number", "1234567"],
         ["National Insurance number", "QQ123456C"],
         ["Email address", "test@email.com"],
+        ["Account number", "12345678"],
+        ["Sort code", "123456"],
```

Comment:
We may want to format the sort code in the standard way when we show it back to the user. However it wasn't an AC on this card so we can probably leave it for a future iteration

Reply:
Yep, I think we have this issue to tackle for a lot of the fields.